### ISSUE TITLE
MIDI generated is just one note repeated 

### ISSUE THREAD

**Issue by Stefan-Hopman** [Skuldur/Classical-Piano-Composer#51]
Hello, 

First of all, thank you for putting this repo together! I am running into an issue where my generated MIDI is just one note repeated. I made two change to the model.  I changed LSTM to CuDNNLSTM and added dropout layers in between because CuDNNLSTM has no inbuilt dropout layers. Does anyone know why this is happening? My model is 
 ```def create_network_generate(network_input, n_vocab):
   model = Sequential()
   model.add(CuDNNLSTM(
        512,
        input_shape=(network_input.shape[1], network_input.shape[2]),
        return_sequences=True
    ))
    model.add(Dropout(0.2))
    model.add(CuDNNLSTM(512, return_sequences=True,))
    model.add(Dropout(0.2))
    model.add(CuDNNLSTM(512))
    model.add(Dropout(0.2))
    model.add(BatchNorm())
    model.add(Dropout(0.3))
    model.add(Dense(256))
    model.add(Activation('relu'))
    model.add(BatchNorm())
    model.add(Dropout(0.3))
    model.add(Dense(n_vocab))
    model.add(Activation('softmax'))
    model.compile(loss='categorical_crossentropy', optimizer='rmsprop')
    return model 
```

<img width="603" alt="Screen Shot 2022-05-03 at 11 52 45 AM" src="https://user-images.githubusercontent.com/85311565/166489241-39bfa694-84a6-45c1-b246-f20576bb9159.png">

**Comment by DiscoLucas** [comment]
Hey, I'm experiencing the same issue, did you ever find a solution?

**Comment by xngln24** [comment]
Any solution found yet? I have the same problem.

**Comment by skrinsky** [comment]
hhheeeeeellllllppppp usssssss

**Comment by skrinsky** [comment]
please <3 :):) AHHHHHHHH
